docs/atomicapp_lifecycle.md

```diff
@@ -0,0 +1,58 @@
+Atomicapp Lifecycle Definition
```

Comment:
no, maybe, we have define that we call it 'Atomic App' if we are talking about the project in a more marketingish context and 'atomicapp' if it comes to source and technical stuff...